packages/shared/src/stac/index.ts

```diff
@@ -1,5 +1,6 @@
-export const StacVersion = '1.0.0';
+export const StacVersion = '1.0.0-beta.2';
 export const StacLicense = 'CC-BY-4.0';
+export const StacBaseMapsExtension = 'https://basemaps.linz.govt.nz/json-schema/basemaps-extension/1.0/schema.json';
```

Comment:
is this URL a standard for this schema?

How is the file deployed?

Reply:
There is no standard for vendor extensions. I am trying to use a somewhat canonical path.

The file should be deployed as part of the landing package (assuming landing deploys everything under `static`)

Reply:
I see that the deploy does not deploy everything under static. How is `favicon-16x16.png` etc getting deployed?

Reply:
>  I see that the deploy does not deploy everything under static. How is favicon-16x16.png etc getting deployed?

Not too sure maybe I broke it a while back? possibly should be fixed :)

Reply:
It is deploying every file under dist but not descending directories; now it does.

Reply:
Actually only deploy copies everything under `dist`. Bundle needs to be modified to copy everything from `static` to `dist`

Reply:
`bundle` now recursively copies `static` files (not `index.css` etc) to `dist`